v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent INSERT, UPDATE, UPSERT, and DELETE statements within a single transaction. 
+
+To execute statements in parallel, use the `RETURNING NOTHING` clause for the independent SQL statements.
```

Comment:
I'm not sure there's a great answer here. Typically, we use `clause` when values need to be provided following an important keyword like `ORDER BY` or `WHERE`. In places where `RETURNING` is followed by values, therefore, it makes sense to refer to the `RETURNING` clause. But here, there's nothing but keywords. `suffix` is precise in that these keywords must always be at the end of the statement. I'd suggest we just go with that for now.